### PR TITLE
Ambari documentation.

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build](http://builds.cask.co/browse/CDAP-DOB28-1)

### CODE REVIEW

File: cdap-docs/integrations/source/ambari/index.rst [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+Adding the CDAP Service to Ambari
+=================================
+To install CDAP on a cluster managed by Ambari, we have provided packages for
+RHEL-compatible and Ubuntu systems, which are installed onto the Ambari management server.
+These packages add CDAP to the list of available services which Ambari can install. 
+
+To install the ``cdap-ambari-service`` package, first add the appropriate CDAP repository
+to your system’s package manager by following :ref:`this procedure for installing a Cask
+repository <install-preparing-the-cluster>` on your Ambari server.
```
Comment (wolf31o2) [inline cdap-docs/integrations/source/ambari/index.rst]: This, by default, will show the 3.0 repository stuff, since it's the 3.0 docs, right?

**Comment by wolf31o2** [comment]
One minor question, but I suspect I already know the answer. :+1: LGTM

**Comment by awholegunch** [comment]
It passed Quick Build, except I forgot that Quick Build doesn't work with the older releases...fixing...

**Comment by awholegunch** [comment]
Passed [Quick Build (pre 3.2)](http://builds.cask.co/browse/CDAP-DQBP30-1)
[HTML Files](http://builds.cask.co/artifact/CDAP-DQBP30/shared/build-1/Docs-HTML/3.0.5/en/integrations/ambari/index.html)
[Package Managers](http://builds.cask.co/artifact/CDAP-DQBP30/shared/build-1/Docs-HTML/3.0.5/en/admin-manual/installation/installation.html#preparing-package-managers)
Is that what you were looking for?

**Comment by wolf31o2** [comment]
:+1: LGTM

**Pull request merged**